Remove direct/indirect import and export distinction, and simplify text. (#85)
ietf-tapswg/draft-ietf-taps-transport-security: draft-ietf-taps-transport-security.md

  - tcpcrypt

- Pre-Shared Key Import:
Either the handshake protocol or the application directly can supply pre-shared keys for use
in encrypting (and authenticating) connections to a peer.
  - TLS
  - DTLS
  - QUIC
  - ESP
  - OpenVPN
  - tcpcrypt
  - MinimalT

>   - WireGuard

Reviewer: Let's add IKEv2 here too

  - tcpcrypt

- Pre-Shared Key Import:
Either the handshake protocol or the application directly can supply pre-shared keys for use

> in encrypting (and authenticating) connections to a peer.

Reviewer 2: Maybe let's substitute "connections" here with "communications" or "data sent"? Not all of these protocols are connection-oriented.

Reviewer: ```suggestion
in encrypting (and authenticating) communication with a peer.
```
